### PR TITLE
Fix Demote Locking/banning users

### PR DESCRIPTION
**Pull request by DanielCranham**
According to resource monitor Locking/banning is implied that it will demote both the target users global and room auth, where as currently it only demotes the target user from roomauth, this commit fixes it to demote the user from both room and global auth.

### CODE REVIEW

**Comment by kotarou3** [comment]
Does the current code for it not work?
https://github.com/Zarel/Pokemon-Showdown/blob/master/users.js#L1159

If it doesn't, fix it there rather than where you've done it right now

File: commands.js [hunk 1]
```diff
@@ -2207,5 +2209,4 @@ var commands = exports.commands = {
 			}
 		}
 	}
-
```
Comment (kotarou3) [inline commands.js]: Also, avoid unrelated whitespace changes being commited with something else

**Comment by DanielCranham** [comment]
Alright, I used the users.js implentation this time after taking kota's advice.

**Comment by Zarel** [comment]
This is still wrong.

**Comment by Zarel** [comment]
This actually confirms a user, which is the opposite of what `deconfirm` is supposed to do.

**Comment by Zarel** [comment]
Done in 169a9756ad7
